src/Sylius/Bundle/ResourceBundle/Controller/ResourceController.php

```diff
@@ -550,4 +531,87 @@ protected function findOr404(RequestConfiguration $configuration)
 
         return $resource;
     }
+
+    /**
+     * @param RequestConfiguration $configuration
+     * @param ResourceInterface $resource
+     *
+     * @return array
+     */
+    protected function getShowTemplateData(RequestConfiguration $configuration, ResourceInterface $resource)
+    {
+        return [
+            'configuration' => $configuration,
+            'metadata' => $this->metadata,
+            'resource' => $resource,
+            $this->metadata->getName() => $resource,
+        ];
+    }
+
+    /**
+     * @param RequestConfiguration $configuration
+     * @param mixed $resources
```

Comment:
No :( As I noticed, there could be passed array or ``ResourceGridView`` and God knows what else... 😢